pkg/planner/core/operator/physicalop/base_physical_plan.go

```diff
@@ -404,3 +404,24 @@ func NewBasePhysicalPlan(ctx base.PlanContext, tp string, self base.PhysicalPlan
 		Self: self,
 	}
 }
+
+func admitIndexJoinProps(children []*property.PhysicalProperty, prop *property.PhysicalProperty) []*property.PhysicalProperty {
```

Comment:
Looks you didn't delete this function in `exhaust_physical_plans.go`.

Reply:
I have deleted this function.